modules/index.js

```diff
@@ -0,0 +1,51 @@
+import tslib from '../tslib.js';

```

Comment:
Why not add scripts to copy `tslib.es6.js` to `modules/index.js`?
import cjs from mjs will cause warning in angular cli.